### PR TITLE
Dan.constantinescu.day11

### PR DESCRIPTION
**Pull request by aircraft721**

### CODE REVIEW

File: start-from-zero/Javascript/src/api/loginData.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+        const userCredential = document.getElementById('username-login').value;
+        const passwordCredential = document.getElementById('password-login').value;
+        const newUser = new Users(userCredential,passwordCredential);
+
+        localStorage.setItem('newUser',JSON.stringify(newUser));
+
+        const parsedCredentials = JSON.parse(localStorage.getItem('newUser'));
+
+        console.log(parsedCredentials);
```
Comment (s-timofte) [inline start-from-zero/Javascript/src/api/loginData.js:17]: wild console.log spotted

File: start-from-zero/Javascript/src/api/storeData.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            textareaArr.push(element.value);
+        })
+    
+    
+        const radioArr = [];
+        checkbox.forEach(element => {
+            if(element.checked === true){
+                radioArr.push(element.value)
+            }else{
+                radioArr.push();
```
Comment (s-timofte) [inline start-from-zero/Javascript/src/api/storeData.js:33]: do you need the else branch at all here?

File: start-from-zero/Javascript/src/data/data.js [hunk 3]
```diff
@@ -86,7 +86,7 @@ function FieldsetData(){
                         span: 'Classes',
                         name: 'classes',
                         optionValue: ['0', '1', '2', '3'],
-                        dropdown: ['Javascript', 'Java', 'Python', 'Haskell'],
+                        dropdown: ['xbox', 'Java', 'Python', 'Haskell'],
```
Comment (s-timofte) [inline start-from-zero/Javascript/src/data/data.js:89]: can't wait to have my first xbox interview

File: start-from-zero/Javascript/src/views/evaluation/evaluation.js [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
-            level:'Master'
-        },
-        {
-            name:'Fiodor D',
-            language: 'Russian',
-            level:'Master'
-        },
-    ];
+    const rows = JSON.parse(localStorage.getItem('evaluations'));
+    // [
```
Comment (s-timofte) [inline start-from-zero/Javascript/src/views/evaluation/evaluation.js:4]: please remove commented code you no longer need

File: start-from-zero/Javascript/src/api/storeData.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+        const username = document.getElementById('username-id').value;
+        const password = document.getElementById('password-id').value;
+        const calendar = document.getElementById('calendar-id').value;
+        const checkbox = document.querySelectorAll('.input-checkbox');
+        const textarea = document.querySelectorAll('textarea');
+        const legend = document.querySelectorAll('legend');
+        const select = document.querySelectorAll('select');
+        
+        const textareaArr = [];
```
Comment (s-timofte) [inline start-from-zero/Javascript/src/api/storeData.js:22]: I would much rather that we don't couple content with the element index; if I add another textarea you will have to change the whole data grabber process; any better way you can think of?

**Review by s-timofte — APPROVED** [review]

File: start-from-zero/Javascript/src/data/data.js (hunk 3 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/data/data.js:89]: :))

File: start-from-zero/Javascript/src/api/storeData.js (hunk 5 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/api/storeData.js:22]: Yes. I solved this.

File: start-from-zero/Javascript/src/views/evaluation/evaluation.js (hunk 4 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/views/evaluation/evaluation.js:4]: Ok. Sorry about this.

File: start-from-zero/Javascript/src/api/loginData.js (hunk 1 above)
Comment (aircraft721) [reply start-from-zero/Javascript/src/api/loginData.js:17]: I totally forgot about that :))

**Pull request merged**